libsql-server/src/database/mod.rs

```diff
@@ -184,7 +184,7 @@ impl Database {
         match self {
             Database::Primary(p) => Some(p.wal_wrapper.wrapper().logger()),
             Database::Replica(_) => None,
-            Database::Schema(s) => Some(s.wal_wrapper.wrapper().logger()),
+            Database::Schema(s) => Some(s.wal_wrapper.as_ref().unwrap().wrapper().logger()),
```

Comment:
is this unwrap dangerous? Maybe we should make this an except?

Reply:
It's not dangerous because we never take it out, we could even make it type safe by adding a generic additional extra-data field set to the wal wrapper

We can add an expect no problem though